tests/integration_test/grpc_streaming_test.go

```diff
@@ -29,6 +29,8 @@ import (
 )
 
 func TestGRPCStreaming(t *testing.T) {
+	//TODO(taoxuy): add t.parallel()
```

Comment:
As discussed offline, these probably fail because they make conflicting files. Let's try to write the files with different names to `/tmp`.